add a description for configuration item query-feedback-limit (#3515)
pingcap/docs: releases/release-3.0.17.md

+ TiDB

>     - Modify the default value of configuration item from 1024 to 512, ease the impact of statistics feedback on the cluster [#18770](https://github.com/pingcap/tidb/pull/18770)

Reviewer: ```suggestion
    - Modify the default value of the `query-feedback-limit` configuration item from 1024 to 512, which eases the impact of statistics feedback on the cluster [#18770](https://github.com/pingcap/tidb/pull/18770)
```

>     - Modify the default value of the `query-feedback-limit` configuration item from 1024 to 512, which eases the impact of statistics feedback on the cluster [#18770](https://github.com/pingcap/tidb/pull/18770)

Reviewer 2: ```suggestion
    - Decrease the default value of the `query-feedback-limit` configuration item from 1024 to 512, and improve the statistics feedback mechanism to ease its impact on the cluster [#18770](https://github.com/pingcap/tidb/pull/18770)
```
